src/qtt/algorithms/functions.py

```diff
@@ -471,3 +471,60 @@ def logistic(x, x0=0, alpha=1):
     """
     f = 1 / (1 + np.exp(-2 * alpha * (x - x0)))
     return f
+
+
+def raised_cosine_frequency_domain(f: np.ndarray, beta: float, T: float) -> np.ndarray:
+    """ Raised cosine frequency domain function
+
+    See https://en.wikipedia.org/wiki/Raised-cosine_filter
+
+    Args:
+        t: Frequency
+        beta: Roll-off factor
+        T: Symbol period
+
+    Returns:
+        Calculated values of the raised cosine
+    """
+    two_T = 2*T
+    if beta == 0:
+        return np.abs(f) < 1/(two_T)
+    ww = np.abs(f)-(1-beta)/(two_T)
+    value = .5*(1+np.cos((np.pi*T/beta)*ww))
+    idx = np.abs(f) <= (1-beta)/two_T
+    value[idx] = 1
+    idx = np.abs(f) >= (1+beta)/two_T
+    value[idx] = 0
+    return value
+
+
+def raised_cosine(t: np.ndarray, beta: float, T: float) -> np.ndarray:
```

Comment:
Arguments t, beta and T are not descriptive to me. Especially t and T as parameters of one method. I understand they come from the formulas on wikipedia but we should create descriptive names. Also python doesn't prescribe capitals in argument names.